integration/spark-common/src/main/java/org/apache/carbondata/spark/merger/CarbonCompactionUtil.java

```diff
@@ -142,20 +142,15 @@ private static void groupCorrespodingInfoBasedOnTask(TableBlockInfo info,
    * @return
    */
   public static boolean isCompactionRequiredForTable(String metaFolderPath) {
-    String minorCompactionStatusFile = metaFolderPath + CarbonCommonConstants.FILE_SEPARATOR
-        + CarbonCommonConstants.minorCompactionRequiredFile;
-
-    String majorCompactionStatusFile = metaFolderPath + CarbonCommonConstants.FILE_SEPARATOR
-        + CarbonCommonConstants.majorCompactionRequiredFile;
+    String statusFile = metaFolderPath + CarbonCommonConstants.FILE_SEPARATOR;
     try {
-      if (FileFactory.isFileExist(minorCompactionStatusFile,
-          FileFactory.getFileType(minorCompactionStatusFile)) || FileFactory
-          .isFileExist(majorCompactionStatusFile,
-              FileFactory.getFileType(majorCompactionStatusFile))) {
+      if (FileFactory.isFileExist(statusFile + CarbonCommonConstants.minorCompactionRequiredFile,
+          FileFactory.getFileType(statusFile + CarbonCommonConstants.minorCompactionRequiredFile))
```

Comment:
Used java-code-format-template.xml provided in the dev folder to format the code